packages/contracts-bedrock/contracts/test/Encoding.t.sol

```diff
@@ -5,19 +5,31 @@ import { CommonTest } from "./CommonTest.t.sol";
 import { Types } from "../libraries/Types.sol";
 import { Encoding } from "../libraries/Encoding.sol";
 
-contract Encoding_Test is CommonTest {
+contract Encoding_TestInit is CommonTest {
     function setUp() external {
         _setUp();
     }
+}
+
+contract Encoding_EncodeVersionedNonce_TestFail is Encoding_TestInit {
```

Comment:
Why is this included if there are none?

Reply:
The intention is to express that I've actually thought about whether or not there are any sad path tests that could be written here, and determined that there are none.
I like having it, but I'm not attached to it.